public/src/main/java/com/revenuecat/purchases/EntitlementVerificationMode.kt

```diff
@@ -0,0 +1,43 @@
+package com.revenuecat.purchases
+
+/**
+ * Verification strictness levels for [EntitlementInfo].
+ */
+enum class EntitlementVerificationMode {
+    /**
+     * The SDK will not perform any entitlement verification.
+     */
+    DISABLED,
+
+    /**
+     * Enable entitlement verification.
+     *
+     * If verification fails, this will be indicated with [VerificationResult.FAILED]
```

Comment:
Note that `VerificationResult` does not exist yet. Plan to add it in a future PR.